src/index.html

```diff
@@ -11,6 +11,39 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="post"
+      class="forms"
```

Comment:
```suggestion
      class="form"
```
name classes in singular form.